### PR TITLE
[NCLSUP-799] extend inline property feature to plugins section

### PR DESCRIPTION
**Pull request by rgdoliveira**
Jira: https://issues.redhat.com/browse/NCLSUP-799

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -635,6 +636,7 @@
       <plugin>
         <groupId>org.projectlombok</groupId>
         <artifactId>lombok-maven-plugin</artifactId>
+        <version>${lombokPluginVersion}</version>
```
Comment (rgdoliveira) [inline pom.xml:639]: Unit Test `testInlineProperty` uses this pom.xml file to run the tests, that's why I created the `lombokPluginVersion` property.

File: core/src/main/java/org/commonjava/maven/ext/core/groovy/BaseScriptUtils.java [hunk 2]
```diff
@@ -73,6 +73,15 @@ public void inlineProperty ( Project currentProject, ProjectRef groupArtifact )
                               a.getValue().setVersion(
                                               PropertyResolver.resolvePropertiesUnchecked( getSession(), currentProject.getInheritedList(), a.getValue().getVersion() ) );
                           } );
+            currentProject.getResolvedPlugins( getSession() )
```
Comment (rnc) [inline core/src/main/java/org/commonjava/maven/ext/core/groovy/BaseScriptUtils.java:76]: I wonder if we should cover managed plugins as well as plugins (given that the function currently covers dependencies and managed dependencies) ?

Comment (rgdoliveira) [reply core/src/main/java/org/commonjava/maven/ext/core/groovy/BaseScriptUtils.java:76]: Good point, I will try to add managed plugins as well.

Comment (rgdoliveira) [reply core/src/main/java/org/commonjava/maven/ext/core/groovy/BaseScriptUtils.java:85]: @rnc I implemented this change and pushed a new commit to the PR.

**Review by rnc — APPROVED** [review]

**Comment by rgdoliveira** [comment]
It seems both CI for maven 3.3.9 and 3.5.4 failed due to time out when downloading artifacts.

**Comment by rnc** [comment]
Thanks for the PR!

**Pull request merged**